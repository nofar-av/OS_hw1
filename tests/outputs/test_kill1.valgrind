==28441== Memcheck, a memory error detector
==28441== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28441== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28441== Command: /home/omer/OS_wet1/smash
==28441== Parent PID: 28437
==28441== 
==28441== 
==28441== FILE DESCRIPTORS: 4 open at exit.
==28441== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_kill1.valgrind
==28441==    <inherited from parent>
==28441== 
==28441== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_kill1.err
==28441==    <inherited from parent>
==28441== 
==28441== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_kill1.out
==28441==    <inherited from parent>
==28441== 
==28441== Open file descriptor 0:
==28441==    <inherited from parent>
==28441== 
==28441== 
==28441== HEAP SUMMARY:
==28441==     in use at exit: 0 bytes in 0 blocks
==28441==   total heap usage: 49 allocs, 49 frees, 82,620 bytes allocated
==28441== 
==28441== All heap blocks were freed -- no leaks are possible
==28441== 
==28441== For counts of detected and suppressed errors, rerun with: -v
==28441== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
