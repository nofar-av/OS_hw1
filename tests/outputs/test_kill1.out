smash> smash> smash> signal number 1 was sent to pid 28633
smash> received signal number: 1
smash: got ctrl-Z
